ViennaAdvantage/Areas/VA012/Models/BankStatementModel.cs

```diff
@@ -4920,6 +5086,22 @@ ELSE CCR1.DIVIDERATE
                         //}
                         //end pratap
                     }
+                    //used new query to get the C_Currency_ID
+                    _sql = @"SELECT UNIQUE INV.C_Currency_ID
```

Comment:
make it Distinct
Unique will not work in postgreSQL

Reply:
Done.